src/components/Explorer/fields.js

```diff
@@ -499,7 +499,7 @@ ${props.player && props.player.value ? '' : 'AND player_matches.account_id < pla
       const laneRoleId = Number(str.substring('lane_role_'.length));
       return { text: strings[str], value: laneRoleId, key: String(laneRoleId) };
     }),
-    isTi8Team: [{ text: 'Yes', value: true, key: 'true' }],
+    isTi9Team: [{ text: 'Yes', value: true, key: 'true' }],
```

Comment:
This is fine, but I'm thinking for next year we could probably just make this isTiTeam so there's not so much code to change.